### DISCUSSION TITLE
Can't change audio default level to 100%

### DISCUSSION THREAD

**Discussion by Gadjet5** [G-a-v-r-o-c-h-e/PIpod-Nano#4] [General]
Hi firstly thanks heaps for putting this all together. Super cool!
I got everything working fine but would love to be able to have the volume default to 100% on boot. 
I tried modifying the autoplay.sh line 9: mpc volume 1  to  mpc volume 100 but this had no affect.. 
Any help would be super appreciated
Joe

**Comment by PetitPrinc3** [comment]
Hi, thanks for your interest in the project.
First, (to be sure) did you edit the autoplay.sh in "/usr/share/PIPodScripts" ?
If so, could you try with any other value, see if it changes anything ?
If you still have no luck making it work, I suggest you look at the existing issue where a "fix" is provided. (Please use it as a last resort)
Keep me updated !

**Reply by Gadjet5** [reply]
Hey thanks and sorry for the late reply! I followed the fix but couldn't get it working then went back and modified the autoplay script again and it worked!... perhaps I had not edited it with sudo the first time... Thanks again for your help

**Reply by PetitPrinc3** [reply]
Great ! Thanks for letting me know 😉

**Reply by gyuchulm** [reply]
you can adjust the volume in /usr/share/PIPodScrips you should edit the autoplay.sh It was a bit complex for me because in the documents also has a autoplay.sh but you should check the place!
